Userland/Utilities/image.cpp

```diff
@@ -16,18 +16,37 @@
 #include <LibGfx/ImageFormats/QOIWriter.h>
 
 struct LoadedImage {
-    RefPtr<Gfx::Bitmap> bitmap;
+    Gfx::NaturalFrameFormat internal_format;
+    AK::Variant<RefPtr<Gfx::Bitmap>, RefPtr<Gfx::CMYKBitmap>> bitmap;
```

Comment:
Note:
On a short term, we should name that `Variant`.
For the long term, I'm also thinking more and more about a class hierarchy for `Bitmaps` and that will be a `RefPtr<BitmapBase>`